test/user.js

```diff
@@ -0,0 +1,129 @@
+const chai = require('chai'),
+  dictum = require('dictum.js'),
+  server = require('./../app'),
+  User = require('../app/models').User,
+  expect = chai.expect,
+  should = chai.should();
+
+describe('users', () => {
+  describe('/users/ POST', () => {
+    it('should create a new user without problems', done => {
+      chai
+        .request(server)
+        .post('/users/')
+        .send({
+          name: 'sarahi',
+          lastName: 'torres',
+          email: 'sarahi12hg@wolox.com',
+          password: 'woloxwoloA1520'
+        })
+        .then(async res => {
+          res.should.have.status(201);
+          const users = await User.find({
+            where: {
+              name: 'sarahi',
+              lastName: 'torres',
+              email: 'sarahi12hg@wolox.com'
+            }
+          });
+
+          expect(users).to.be.a('object');
+          expect(users.password).to.not.equal('woloxwolox1520');
```

Comment:
What are you checking here ?

Reply:
I check that the password is saved encrypted,should I remove it?